src/test/java/com/productdock/rbc2024/mapper/bookmapper/BookMapperShould.java

```diff
@@ -29,4 +29,41 @@ void convertModelToBookDetailsDto() {
         assertEquals(expected, bookDetailsDto);
     }
 
+    @Test
+    void convertModelToBookDto(){
+
```

Comment:
If the code is more readable for you with this empty line, leave like that. If not, you can remove it.